lib/team.js

```diff
@@ -0,0 +1,93 @@
+let ChatMessage
+let MessageBuilder
+
+module.exports = loader
+
+function loader (mcVersion) {
+  ChatMessage = require('prismarine-chat')(mcVersion)
+  MessageBuilder = ChatMessage.MessageBuilder
+  return Team
+}
+
+function colorString (color) {
```

Comment:
could you check whether prismarine-chat could be used to do that ? I feel we already have that info there

Reply:
while we have the data, it's not accessible currently